packages/babel-preset-react-app/index.js

```diff
@@ -14,7 +14,10 @@ const plugins = [
     // class { handleClick = () => { } }
     require.resolve('babel-plugin-transform-class-properties'),
     // { ...todo, completed: true }
-    require.resolve('babel-plugin-transform-object-rest-spread'),
+    [require.resolve('babel-plugin-transform-object-rest-spread'), {
+      // Use Object.assign directly, instead of extends helper
```

Comment:
Let's add a note that this requires `Object.assign` polyfill, and do it in README of the plugin too.